index.bs

```diff
@@ -124,7 +124,9 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
 
 A variety of additional use cases and configurations are also possible, including (but not limited to):
 
-- User navigates to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
+- A user navigates to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
+
+- A user obtains an discreet, cross-platform authenticator, such as a "fob" with USB or USB+NFC/BLE connectivity options, loads example.com in their browser on a laptop or phone, and is guided though a flow to create and register a credential on the fob. 
```

Comment:
I assume you mean discrete (i.e. separate) rather than discreet (i.e. cautious or understated) here.